Cubical/Axiom/Choice.agda

```diff
@@ -0,0 +1,58 @@
+{-# OPTIONS --safe #-}
+
```

Comment:
Maybe a header would be good for this particular file summarizing what it contains?